docs/source/user_documentation/user_documentation.rst

```diff
@@ -125,6 +125,25 @@ SnowShu will report details of the created replica once completed.
 
 .. image:: /../assets/completed_replica.png 
 
+Creating An Incremental Replica
+------------------
+
+There's a simple way to rebuild an existing replica in case the ``replica.yml`` file has been changed. Instead of building a brand new replica, you can apply changes to the existing one.
+Incremental replica creates relations and loads data only for new entries found in ``replica.yml`` file, which are not already present in target replica image.
+The target for the incremental replica is actually a docker image name, which you can find by typing: 
+
+>>> docker images
+
+It usually starts with ``snowshu_replica_``. So for a replica named *hamburger-sandwich*:
+
+>>> snowshu create -i snowshu_replica_hamburger-sandwich
+
+or
+
+>>> snowshu create --incremental snowshu_replica_hamburger-sandwich
```

Comment:
Do the images need to include the tag for this option? `snowshu_replica_hamburger-sandwich:latest`

Or does it default to `latest` when no version is present? I vaguely remember it requiring the version. If so we should document it either way (i.e. defaults to `latest` or make sure the version is included)

Reply:
Hello! Fixed. Please, check!